specification/vmware/Microsoft.AVS.Management/routes.tsp

```diff
@@ -315,85 +315,72 @@ interface GlobalReachConnections {
   "globalReachConnection"
 );
 
-@armResourceOperations
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-interface-requires-decorator"
+#suppress "@azure-tools/typespec-azure-resource-manager/no-resource-delete-operation"
 interface WorkloadNetworks {
   #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
   @operationId("WorkloadNetworks_Get")
+  @tag("WorkloadNetworks")
+  @autoRoute
   get is ArmResourceRead<WorkloadNetwork>;
 
   #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
   @operationId("WorkloadNetworks_List")
+  @tag("WorkloadNetworks")
+  @autoRoute
   list is ArmResourceListByParent<
     WorkloadNetwork,
     Response = ArmResponse<ResourceList<WorkloadNetwork>>
   >;
-}
```

Comment:
Should not change the interface

Reply:
Hi @markcowl 

I actually made an attempt to do the grouping in client.tsp

However, I got a problem of `@singleton` resource. The auto-route of this kind of resource only work when the operation is under `@armProviderNamespace` decorated namespace.

Detail here https://github.com/Azure/azure-rest-api-specs/pull/29308#discussion_r1687630775